config/locales/en.yml

```diff
@@ -1124,6 +1124,7 @@ en:
       recipient_details_heading: Where should the complaint response be sent?
       requester_heading: Who is making the complaint?
       request_details_heading: What is the date on the complaint?
+      requested_info_heading: Is there other information relevant for this complaint?
```

Comment:
This would maybe read a little clearer as " Is there other information relevant to this complaint?"